lib/router/mbaas.js

```diff
@@ -86,7 +89,27 @@ function initRouter(mediator) {
   return router;
 }
 
+/**
+* Function to trim the authentication response to remove certain fields from being sent.
+* By default, the password will be removed from the response.
+* @param authResponse {object} - the untrimmed auth response
+* @param exclusionList {array} - the array of field names to remove from the authentication response
+* @return authResponse {object} - the trimmed authentication response
+*/
+function trimAuthResponse(authResponse, exclusionList) {
```

Comment:
@vchepeli It's not changed. https://lodash.com/docs/4.17.2#omit will return  a different object.